test/universal/DoubleNestedMultisigBuilder.t.sol

```diff
@@ -79,14 +79,20 @@ contract DoubleNestedMultisigBuilderTest is Test, DoubleNestedMultisigBuilder {
 
     function test_sign_double_nested_safe1() external {
         vm.recordLogs();
-        sign(safe1, safe3);
+        bytes memory txData = abi.encodeCall(DoubleNestedMultisigBuilder.sign, (safe1, safe3));
```

Comment:
A potentially nicer syntax would be `DoubleNestedMultisigBuilder(address(this)).sign(safe1, safe3);`